From 04d467ced9ce21ee2e34464c00a2d1382d2ded00 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 17 Apr 2016 17:34:51 +0300
Subject: [PATCH 24/38] ARM: i.MX6UL: dts: update cl-som-imx6ul model string

Update cl-som-imx6ul model string in order to
meet the CompuLab ATP machine name requirements.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dts     | 2 +-
 arch/arm/boot/dts/imx6ul-sbc-imx6ul-wilink.dts | 2 +-
 arch/arm/boot/dts/imx6ul-sbc-imx6ul.dts        | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dts b/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dts
index 1138af2..3cfdefb 100644
--- a/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dts
+++ b/arch/arm/boot/dts/imx6ul-cl-som-imx6ul.dts
@@ -11,6 +11,6 @@
 #include "imx6ul-cl-som-imx6ul.dtsi"
 
 / {
-	model = "CompuLab i.MX6 UltraLite SoM";
+	model = "CompuLab CL-SOM-iMX6UL";
 	compatible = "fsl,imx6ul", "compulab,cl-som-imx6ul";
 };
diff --git a/arch/arm/boot/dts/imx6ul-sbc-imx6ul-wilink.dts b/arch/arm/boot/dts/imx6ul-sbc-imx6ul-wilink.dts
index 8301d37..bdef44c 100644
--- a/arch/arm/boot/dts/imx6ul-sbc-imx6ul-wilink.dts
+++ b/arch/arm/boot/dts/imx6ul-sbc-imx6ul-wilink.dts
@@ -6,6 +6,6 @@
 #include "imx6ul-sb-som-imx6ul.dtsi"
 
 / {
-	model = "Compulab iMX6UL on SB-SOM";
+	model = "CompuLab CL-SOM-iMX6UL on SBC-SOM-iMX6UL";
 	compatible = "fsl,imx6ul", "compulab,cl-som-imx6ul", "compulab,sbc-imx6ul";
 };
diff --git a/arch/arm/boot/dts/imx6ul-sbc-imx6ul.dts b/arch/arm/boot/dts/imx6ul-sbc-imx6ul.dts
index 9e97b14..48a939d 100644
--- a/arch/arm/boot/dts/imx6ul-sbc-imx6ul.dts
+++ b/arch/arm/boot/dts/imx6ul-sbc-imx6ul.dts
@@ -6,6 +6,6 @@
 #include "imx6ul-sb-som-imx6ul.dtsi"
 
 / {
-	model = "Compulab iMX6UL on SB-SOM";
+	model = "CompuLab CL-SOM-iMX6UL on SBC-SOM-iMX6UL";
 	compatible = "fsl,imx6ul", "compulab,cl-som-imx6ul", "compulab,sbc-imx6ul";
 };
-- 
1.9.1

